### PR TITLE
feat: Add CLI `start`

### LINKED ISSUE #15: Add CLI: start
- subscribe  and handling relevant events (#3)
- handling data validation requests for data certificate (#6)

### PR DESCRIPTION
**Pull request by H4NLee**
close #15

### CODE REVIEW

**Review by gyuguen — APPROVED** [review]
LGTM

File: cmd/oracled/cmd/start.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+			if err != nil {
+				return fmt.Errorf("failed to create service: %w", err)
+			}
+			defer svc.Close()
+
+			err = svc.StartSubscriptions(
+				oracleevent.NewRegisterOracleEvent(svc),
+			)
+
+			if err != nil {
```
Comment (youngjoon-lee) [inline cmd/oracled/cmd/start.go:39]:
```suggestion
			)
			if err != nil {
```

**Review by youngjoon-lee — APPROVED** [review]
good

File: server/server.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+			WriteTimeout: 15 * time.Second,
+			ReadTimeout:  15 * time.Second,
+		},
+	}
+}
+
+func (srv *Server) Run() error {
+	log.Infof("HTTP server is started: %s", srv.Addr)
+	// TODO: add TLS
+	return srv.ListenAndServe()
```
Comment (youngjoon-lee) [inline server/server.go:32]:
```suggestion
	return srv.ListenAndServe()
```

According to the current plan, TLS will be handled by load balancer of cloud providers (Azure, AWS, ...).

**Review by audtlr24 — APPROVED** [review]
lgtm

**Pull request merged**